code/LeftAndMain.php

```diff
@@ -1681,7 +1663,6 @@ public function currentPageID()
             return $this->getRequest()->param('ID');
         }
 
-        /** @deprecated */
```

Comment:
Could you leave this in - we'll need to separately what the `".currentPage"` functionality was - originally it looks like someone intended for it to be removed

Reply:
Done